chrobalt/build/configs/initialize_variables.gni

```diff
@@ -31,7 +31,7 @@ declare_args() {
 
   using_old_compiler = false
 
-  build_with_separate_cobalt_toolchain = true
+  build_with_separate_cobalt_toolchain = target_os == "linux"
```

Comment:
Perhaps `target_os != "android"` here, as Android is the "special" carve out we have. But doesn't matter much either way.